modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -0,0 +1,219 @@
+= Guide to Named Examples
+
+If you want to use more than one example for a type, resource type, or trait, you must name the examples. You can also name a single example. When you name examples, you must use the `examples` facet.
+
+To use one example without naming it, you must use the `example` facet.
+
+
+== Naming a single example in the `examples` facet
+
+In the sample files in this section, the `examples` facet names the example `fullName`. The fragment is not declared as a NamedExample fragment because it does not provide a name.
+
+.api.raml
+----
+#%RAML 1.0
+title: test
+types:
+  A:
+    properties:
+      givenName: string
+      familyName: string
+    examples:
+      fullName: !include fragment.raml
+----
+
+.fragment.raml
+----
+#%RAML 1.0
+givenName: ”Chiaki”
+familyName: "Mukai"
+----
+
+
+== Naming a single example in a NamedExample fragment
+
+In the sample files in this section, the `examples` facet does not name the example. The NamedExample fragment names the example `fullName`.
+
+.api.raml
+----
+#%RAML 1.0
+title: test
+types:
+  A:
+    properties:
+      givenName: string
+      familyName: string
+    examples: !include fragment.raml
+----
+
+.fragment.raml
+----
+#%RAML 1.0 NamedExample
+fullName:
+  givenName: ”Chiaki”
+  familyName: "Mukai"
+----
+
+== Naming two examples in an `examples` facet
+
+In the sample files in this section, the `examples` facet names the examples `fullName` and `otherFullName`. The fragments are not declared as NamedExample fragments because they do not provide names.
+
+.api.raml
+----
+#%RAML 1.0
+title: test
+types:
+  A:
+    properties:
+      givenName: string
+      familyName: string
+    examples:
+      fullName: !include fragment1.raml
+      otherFullName: !include fragment2.raml
+----
+
+.fragment1.raml
+----
+#%RAML 1.0
+givenName: ”Chiaki”
+familyName: "Mukai"
+----
+
+.fragment2.raml
+----
+#%RAML 1.0
+givenName: "Kyung-won"
+familyName: "Park"
+----
+
+== Naming two examples in a NamedExample fragment
+
+In the sample files in this section, the `examples` facet does not name the examples.  The NamedExample fragment names the examples `fullName` and `otherFullName`.
```

Comment:
ditto

Do you think that the audience needs an example of every possible scenario? I kind of got the idea after the first couple....